File: Python/Easy/2011_Final_Value_of_Variable_After_Performing_Operations/solution.py
class Solution:
    def finalValueAfterOperations(self, operations: list[str]) -> int:
        opValue = {"--X":-1, "X--":-1, "++X":1, "X++":1}
        x = 0

        for op in operations:
            x += opValue[op]
        
        return x

File: Golang/Easy/2011_Final_Value_of_Variable_After_Performing_Operations/solution.go
package finalvalueofvariableafterperformingoperations

func finalValueAfterOperations(operations []string) int {
	opValue := map[string]int{
		"--X": -1,
		"X--": -1,
		"++X": 1,
		"X++": 1,
	}
	x := 0

	for _, op := range operations {
		x += opValue[op]
	}

	return x
}
